lightning/src/ln/channelmanager.rs

```diff
@@ -3173,8 +3170,11 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 		}
 	}
 
-	/// We force-close the channel without letting our counterparty participate in the shutdown
-	fn block_disconnected(&self, header: &BlockHeader, _: u32) {
+	/// Updates channel state based on a disconnected block.
+	///
+	/// If necessary, the channel may be force-closed without letting the counterparty participate
+	/// in the shutdown.
+	pub fn block_disconnected(&self, header: &BlockHeader, _: u32) {
```

Comment:
Probably worth dropping the dummy parameter now that we don't have to conform to a trait, no?